elifearticle/parse.py

```diff
@@ -412,6 +412,27 @@ def build_self_uri_list(self_uri_list):
     return uri_list
 
 
+def build_clinical_trials(clinical_trials):
+    clinical_trial_list = []
+    for clinical_trial in clinical_trials:
+        clinical_trial_object = ea.ClinicalTrial()
+        attr_map = {
+            'id': 'id',
+            'content-type': 'content_type',
+            'document-id': 'document_id',
+            'document-id-type': 'document_id_type',
+            'source-id': 'source_id',
+            'source-id-type': 'source_id_type',
+            'source-type': 'source_type',
+            'text': 'text',
+            'xlink_href': 'xlink_href',
+        }
+        for dict_key, attr_name in attr_map.items():
+            utils.set_attr_if_value(clinical_trial_object, attr_name, clinical_trial.get(dict_key))
+        clinical_trial_list.append(clinical_trial_object)
```

Comment:
more ammunition for the argument that objects are unnecessary. This function is entirely boilerplate.

Reply:
if we were dealing with simple maps then it could be:

```python
def rename(data, rename_map):
    return {rename_map.get(key, key): value for key, value in data.items()}
rename_map = {'content-type': 'content_type', 'document-id': 'document_id', ...}
return [rename_keys(clinical_trial, rename_map) for data in clinical_trial_list]